app/mixins/message-wall-post-notification.js

```diff
@@ -0,0 +1,44 @@
+import Mixin from '@ember/object/mixin';
+
+export default Mixin.create({
+  /**
+  * Constructs a localized message wall post notification body
+  * @param {Ember.Object} model
+  * @returns {string}
+  */
+  getMessageWallPostBody(model) {
+    const firstReplierName = model.get('latestActors.0.name')
+    const postTitle = this.postTitleMarkup;
+
+    // need this data !!
+    const isOwnWall = firstReplierName === '// check model.uri for username? in Message Wall uri';
+    const postRemoved = '// need check to see if post has been removed';
```

Comment:
My earlier comment on this was wrong. The notification model's `type` field doesn't correspond to the types in the Pandora service. However, there's a notification type `message-wall-post-removed` that should handle removed posts in a different file from this one.